### PR TITLE
nrf: cc310: Change license to Apache-2

### PR DESCRIPTION
**Pull request by sigvartmh**
This fixes the issue: #1624 by changing the license to Apache-2. There isn't a need to have these files nordic licensed.

### CODE REVIEW

**Review by nordicjm — APPROVED** [review]

**Review by de-nordic — APPROVED** [review]

File: ext/nrf/cc310_glue.c [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
- * Copyright (c) 2019 Nordic Semiconductor ASA
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
  *
- * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
```
Comment (d3zd3z) [inline ext/nrf/cc310_glue.c:4]: We do need an SPDX-License-Identifier for the Apache license. In fact, none of the rest of the text is needed (but is up to you).

**Review by d3zd3z — CHANGES_REQUESTED** [review]

File: ext/nrf/cc310_glue.c (hunk 1 above)
Comment (de-nordic) [reply ext/nrf/cc310_glue.c:4]: @sigvartmh Can you address the issue?

Comment (sigvartmh) [reply ext/nrf/cc310_glue.c:4]: Done sorry was a bit late .

Comment (de-nordic) [reply ext/nrf/cc310_glue.c:4]: @d3zd3z Can you take a look again?

**Review by d3zd3z — APPROVED** [review]

**Pull request merged**